app/src/main/res/values/strings.xml

```diff
@@ -232,6 +232,10 @@ limitations under the License.
     <string name="pref_key_myexperiment_pass">pref_myexperimentpass</string>
     <string name="pref_summary_myexperiment_pass">Myexperiment password</string>
 
+    <!--Disclaimer:-->
+    <string name="DisclaimerTitle">Disclaimer:</string>
```

Comment:
@Hiteshgautam01 Please follow this coding guidelines, Here your strings resource name should be 
`disclaimer_title` not `DisclaimerTitle`. Please make the change.